scripts/insert_into_appointments.sql

```diff
@@ -1 +1,36 @@
 -- insert appointments creating connections between doctors and patients
+INSERT INTO appointments(start_at, length, doctor_id, patient_id)
```

Comment:
This will cause an error since the columns `start_at` and `length` don't exist in the `appointments` table.